py_mind_memo/view.py

```
@@ -122,6 +132,20 @@ def _on_canvas_click(self, event):
         cx = self.canvas.canvasx(event.x)
         cy = self.canvas.canvasy(event.y)
 
+        # read_onlyモードでは参照モード操作・ドラッグ移動・参照線操作をブロックする
+        if self.read_only:
+            # 画像クリック（拡大表示）は許可
```

Ruff (0.15.9)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)